stablehlo/tests/interpret_convolution.mlir

```diff
@@ -0,0 +1,79 @@
+// RUN: stablehlo-translate --interpret -split-input-file %s
+
+func.func @convolution_op_test_si64() {
```

Comment:
Is the difference between `si64` and `i64` meaningful here? (I think `i64` in an MLIR type means `si64`. Is this relevant to point out in the test name?)

Reply:
There's a long standing issue #22 to try to converge to one. Long story short, the types were inherited from MHLO when StableHLO was first created. To avoid ambiguity, we explicitly write `si64` in the tests.